src/commands/create-unik.ts

```diff
@@ -85,14 +85,28 @@ export class CreateUnikCommand extends WriteCommand {
         const transactionUrl = `${this.api.getExplorerUrl()}/transaction/${transaction.id}`;
         this.log(`Transaction in explorer: ${transactionUrl}`);
 
+        const awaitConfirmation: number = flags["await-confirmation"];
+        if (awaitConfirmation === 0) {
+            this.info(`Transaction accepted by the network: ${transaction.id}`);
+            this.stop(
+                "Transaction not confirmed yet, still in the pool. Track status of the transaction in the chain explorer.",
+            );
+            return {
+                data: {
+                    id: tokenId,
+                    transaction: transaction.id,
+                },
+            };
+        }
+
```

Comment:
I'm convinced by this implementation. This works, but I don't understand why this is not handled by the `waitTransactionConfirmations` method.

Reply:
Too much rework to do, as the method does a premature return.

Mutualizing this behavior for multiple commands willl need a deep refactoring.